packages/fxa-components/.storybook/webpack.config.js

```diff
@@ -0,0 +1,26 @@
+// https://storybook.js.org/docs/configurations/typescript-config/#setting-up-typescript-to-work-with-storybook-1
+module.exports = ({ config, mode }) => {
+  config.module.rules.push({
+    test: /\.(ts|tsx)$/,
+    loader: require.resolve('babel-loader'),
+    options: {
+      presets: [['react-app', { flow: false, typescript: true }]],
+    },
+  });
+  config.resolve.extensions.push('.ts', '.tsx');
+
+  config.module.rules.push({
+    test: /\.s[ac]ss$/i,
+    use:[
+     // Creates `style` nodes from JS strings
+    'style-loader',
+    // Translates CSS into CommonJS
+    'css-loader',
+    // Compiles Sass to CSS
+    'sass-loader'
+    ],
+  });
+  config.resolve.extensions.push('.scss', '.css');
```

Comment:
Huh, I wonder why we don't need this in payments.

Reply:
This is part of the global webpack config in payments. We don't have one of those in fxa-components